solidstart-tanstackquery-tailwind-modules/src/components/UserDropdown/UserDropdown.tsx

```diff
@@ -0,0 +1,83 @@
+import { A } from '@solidjs/router';
+import { createSignal } from 'solid-js';
+import { ChevronDownIcon } from '../Icons';
+import { clickOutside } from '../../utils/onclickOutside';
+import styles from './UserDropdown.module.css';
+import { useAuth } from '../../auth';
+import { SIGN_OUT_URL } from '../../utils/constants';
+
+interface IProps {
+  image: string;
+  username: string;
+}
+
+const UserDropdown = (props: IProps) => {
+  const [expanded, setExpanded] = createSignal(false);
+
+  const _clickOutside = clickOutside;
+
+  const signOut = () => {
```

Comment:
I believe there is a ticket for sign-out functionality. This shouldn't have been implemented yet